dist/index.js

```diff
@@ -45419,7 +45485,7 @@ const main = async () => {
         process.exit(0);
     }
     catch (error) {
-        core.setFailed(error);
+        core.setFailed(error.stack);
```

Comment:
```suggestion
        core.setFailed(`${error.message}\n{error.stack}`);
```

Reply:
the message is already included as the first line of the stack.